gemfiles/rails_7.0.rb

```diff
@@ -5,5 +5,6 @@
 source "https://rubygems.org"
 gemspec path: ".."
 
-gem "activerecord", "~> 7.0.0.alpha", github: "rails/rails", branch: "main"
-gem "activesupport", "~> 7.0.0.alpha", github: "rails/rails", branch: "main"
+# For some reason, Rails 7.0.0.rc1 is tagged as 7.1.0.alpha
```

Comment:
I believe you could use the branch https://github.com/rails/rails/tree/7-0-stable

Reply:
or just use the stable rc release? https://rubygems.org/gems/rails/versions/7.0.0.rc1

Reply:
@morgoth - That's correct, I could indeed.

Reply:
Now that it is possible, let's use the `7.0.0.rc1` release, instead of git.

Reply:
The 7.0 is out, so we can use that instead.
I believe fixing warning should be handled by https://github.com/binarylogic/authlogic/pull/744 per @jaredbeck decision https://github.com/binarylogic/authlogic/pull/745/files#r766910094

@danlaffan Could you update your PR to just change the gemfile?